roles/pulp_api/tasks/main.yml

```diff
@@ -14,6 +14,17 @@
         # We can also try using seport's ignore_selinux_state variable
         - ansible_facts.selinux.status == "enabled"
 
+    - name: Install pulpcore-api socket file (when it applies)
+      template:
+        src: pulpcore-api.socket.j2
+        dest: /lib/systemd/system/pulpcore-api.socket
+        owner: root
+        group: root
+        mode: 0644
+      become: true
+      notify: Restart pulpcore-api.service
+      when: "'unix:' in pulp_api_bind"
```

Comment:
Systemd sockets can also bind to TCP ports. I'd recommend keeping the installation consistent and always binding via systemd. That way you don't need to ask users if they have a pulpcore-api.socket file or not - it's always there.

Reply:
That would be "socket activation" for pulp, right? I like it.